fhir-swagger-generator/src/main/java/com/ibm/fhir/openapi/generator/FHIROpenApiGenerator.java

```diff
@@ -235,13 +244,8 @@ private static void generateAllInOne(Filter filter) throws Exception {
                     definitionsToAdd.add(resourceModelClass);
 
                     // generate definition for all inner classes inside the top level resources.
-                    for (String innerClassName : getAllResourceInnerClasses()) {
-                        String parentClassName = innerClassName.split("\\$")[0];
-                        if (resourceClassName.equals(parentClassName)) {
-                            Class<?> innerModelClass = Class.forName(RESOURCEPACKAGENAME + "." + innerClassName);
-                            definitionsToAdd.add(innerModelClass);
-                        }
-                    }
+                    Set<String> classFilter = Stream.of(resourceClassName).collect(Collectors.toSet());
```

Comment:
```suggestion
                    Set<String> classFilter = Collections.singleton(resourceClassName);
```

Reply:
just a style/preference thing.  the advantage of the Stream.of pattern is there are instances of this pattern below where there are multiple values passed and so the Stream.of is slightly more consistent.  i'm fine either way.